package.json

```diff
@@ -8,14 +8,15 @@
     "build:drive:mobile": "COZY_APP_SLUG=drive cozy-scripts build --mobile --manifest src/drive/targets/manifest.webapp --src-dir src/drive",
     "watch:drive": "npm run watch:drive:browser",
     "watch:drive:browser": "COZY_APP_SLUG=drive cozy-scripts watch --manifest src/drive/targets/manifest.webapp --build-dir build/drive --src-dir src/drive",
-    "watch:drive:browser:hot": "COZY_APP_SLUG=drive cozy-scripts start  --manifest src/drive/targets/manifest.webapp --build-dir build/drive --src-dir src/drive ",
+    "watch:drive:browser:hot": "COZY_APP_SLUG=drive cozy-scripts start --manifest src/drive/targets/manifest.webapp --build-dir build/drive --src-dir src/drive",
     "watch:drive:mobile": "COZY_APP_SLUG=drive NODE_ENV=mobile:development cozy-scripts watch --manifest src/drive/targets/manifest.webapp --src-dir src/drive",
     "watch:drive:mobile:hot": "COZY_APP_SLUG=drive NODE_ENV=mobile:development  cozy-scripts start  --mobile --manifest src/drive/targets/manifest.webapp --src-dir src/drive ",
     "watch:drive:standalone": "COZY_APP_SLUG=drive NODE_ENV=mobile:development cozy-scripts start  --manifest src/drive/targets/manifest.webapp --src-dir src/drive",
     "build:photos": "npm-run-all --parallel build:photos:browser",
     "build:photos:browser": "COZY_APP_SLUG=photos cozy-scripts build --manifest src/photos/targets/manifest.webapp --build-dir build/photos --src-dir src/photos",
     "watch:photos": "npm run watch:photos:browser",
-    "watch:photos:browser": "COZY_APP_SLUG=photos cozy-scripts start --manifest src/photos/targets/manifest.webapp --build-dir build/photos --src-dir src/photos",
+    "watch:photos:browser": "COZY_APP_SLUG=photos cozy-scripts watch --manifest src/photos/targets/manifest.webapp --build-dir build/photos --src-dir src/photos",
+    "watch:photos:browser:hot": "COZY_APP_SLUG=photos cozy-scripts start --manifest src/photos/targets/manifest.webapp --build-dir build/photos --src-dir src/photos",
```

Comment:
Nit: Hot reloading should be the default and not the exception now. In banks we have a "cold" script. But I think the best would be to use the verb start as in cozy-create-app and create-react-app. start for hot, watch for cold.

Reply:
Ok pour partir sur des `yarn start:drive` et `yarn watch:drive` pour le hot/cold 👍